backend/src/main/resources/log4j2.xml

```diff
@@ -3,6 +3,9 @@
     <Appenders>
         <Console name="ConsoleJSONAppender" target="SYSTEM_OUT">
             <JsonLayout complete="false" compact="true" eventEol="true" stacktraceAsString="true">
+                <!-- Doc link: https://logging.apache.org/log4j/2.x/manual/lookups.html -->
+                <KeyValuePair key="hostName" value="${env:hostName:-}"/>
+                <KeyValuePair key="kubernetes.podIP" value="${k8s:podIp:-}"/>
```

Comment:
this is for Kubernetes pod IP.  Doc link: https://logging.apache.org/log4j/2.x/manual/lookups.html if we want to add more custom lookups for the Kubernetes.